src/Swashbuckle.AspNetCore.SwaggerGen/SwaggerGenerator/SwaggerGeneratorOptions.cs

```diff
@@ -61,12 +65,20 @@ public SwaggerGeneratorOptions()
 
         public IList<IParameterFilter> ParameterFilters { get; set; }
 
-        public List<IRequestBodyFilter> RequestBodyFilters { get; set; }
+        public List<IParameterAsyncFilter> ParameterAsyncFilters { get; set; }
+
+        public IList<IRequestBodyFilter> RequestBodyFilters { get; set; }
```

Comment:
These two are the wrong way around:

```suggestion
        public IList<IParameterAsyncFilter> ParameterAsyncFilters { get; set; }

        public List<IRequestBodyFilter> RequestBodyFilters { get; set; }
```